framework/auth/__init__.py

```diff
@@ -48,6 +49,7 @@ def logout():
             del session.data[key]
         except KeyError:
             pass
+    Session.remove(Q('_id', 'eq', session._id))
```

Comment:
I don't understand why it's necessary to remove the session from the DB either. @alexschiller, can you clarify?